applications/StructuralMechanicsApplication/tests/cl_test/SimpleSmallDeformationPlasticity/plasticity_j2_cube_test_parameters.json

```diff
@@ -0,0 +1,150 @@
+{
+    "problem_data"             : {
+        "problem_name"    : "plasticity_j2_cube_test",
+        "parallel_type"   : "OpenMP",
+        "start_time"      : 0.0,
+        "end_time"        : 2.0,
+        "echo_level"      : 0
+    },
+    "solver_settings"          : {
+        "solver_type"                        : "Static",
+        "echo_level"                         : 0,
+        "domain_size"                        : 3,
+        "model_part_name"                    : "Structure",
+        "analysis_type"                      : "non_linear",
+        "model_import_settings"              : {
+            "input_type"     : "mdpa",
+            "input_filename" : "cl_test/SimpleSmallDeformationPlasticity/plasticity_j2_cube_test"
+        },
+        "material_import_settings"           : {
+            "materials_filename" : "cl_test/SimpleSmallDeformationPlasticity/plasticity_j2_cube_test_materials.json"
+        },
+        "time_stepping"                      : {
+            "time_step" : 0.2
+        },
+        "line_search"                        : false,
+        "convergence_criterion"              : "residual_criterion",
+        "displacement_relative_tolerance"    : 0.0001,
+        "displacement_absolute_tolerance"    : 1e-9,
+        "residual_relative_tolerance"        : 0.0001,
+        "residual_absolute_tolerance"        : 1e-9,
+        "max_iteration"                      : 10,
+        "problem_domain_sub_model_part_list" : ["Parts_Parts_Auto1"],
+        "processes_sub_model_part_list"      : ["Parts_Parts_Auto1","DISPLACEMENT_Displacement_SymXY", "DISPLACEMENT_Displacement_SymXZ", "DISPLACEMENT_Displacement_SymYZ", "DISPLACEMENT_Displacement_Auto2", "DISPLACEMENT_Displacement_Auto3"],
+        "rotation_dofs"                      : false
+    },
+    "processes" : {
+        "constraints_process_list" : [{
+            "python_module" : "assign_vector_variable_process",
+            "kratos_module" : "KratosMultiphysics",
+            "Parameters"    : {
+                "model_part_name" : "DISPLACEMENT_Displacement_SymXY",

```

Comment:
can you please add the main-model-part name to all submodelpart-names?

This way we will have to do less changes once the `Model` removes the deprecated functions

Reply:
ping

Reply:
DOne